src/main/resources/pom-servers.xml

```diff
@@ -574,6 +574,76 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>liberty-2.2</id>
+
+            <properties>
```

Comment:
Why do you have to duplicate so much stuff here? Is it possible to create a procedure like and share the same configuration and <build>?